docker/docker-compose-multiclusters.yml

```diff
@@ -0,0 +1,72 @@
+version: '3'
+services:
+  cassandra:
+    image: cassandra:3.11
+    ports:
+      - "9042:9042"
+  statsd:
+    image: graphiteapp/graphite-statsd
+    ports:
+      - "8080:80"
+      - "2003:2003"
+      - "8125:8125"
+      - "8126:8126"
+  cadence:
+    image: ubercadence/server:master-auto-setup
+    ports:
+     - "7933:7933"
+     - "7934:7934"
+     - "7935:7935"
+     - "7939:7939"
+    environment:
+      - "CASSANDRA_SEEDS=cassandra"
+      - "STATSD_ENDPOINT=statsd:8125"
+      - "DYNAMIC_CONFIG_FILE_PATH=config/dynamicconfig/development.yaml"
+      - "ENABLE_GLOBAL_DOMAIN=true"
+      - "KEYSPACE=cadence_active"
+      - "VISIBILITY_KEYSPACE=cadence_visibility_active"
+      - "STATSD_FRONTEND_PREFIX=cadence-frontend-active"
+      - "STATSD_MATCHING_PREFIX=cadence-matching-active"
+      - "STATSD_HISTORY_PREFIX=cadence-history-active"
+      - "STATSD_WORKER_PREFIX=cadence-worker-active"
+    depends_on:
+      - cassandra
+      - statsd
+  cadence-standby:
+    image: ubercadence/server:master-auto-setup
+    ports:
+     - "7943:7933"
+     - "7944:7934"
+     - "7945:7935"
+     - "7949:7939"
```

Comment:
nit: Can we make the port number consistent with those in `development_standby.yaml`? Makes it a bit easier when switching back and forth between different setup.
```
  bootstrapHosts: [ "127.0.0.1:8933", "127.0.0.1:8934", "127.0.0.1:8935", "127.0.0.1:8940" ]
```